components/Navigation/index.js

```diff
@@ -0,0 +1,40 @@
+import { styled } from "styled-components";
+import Link from "next/link";
+import { device } from "@/utils/globalValues";
+
+export default function Navigation() {
+  return (
+    <StyledNavigation>
+      <li className="inherit-background-color">
+        <StyledNavigationLink href="/words">Words</StyledNavigationLink>
+      </li>
+      <li className="inherit-background-color">
+        <StyledNavigationLink href="/add">Add</StyledNavigationLink>
+      </li>
+    </StyledNavigation>
```

Comment:
You should probably nest your elements like so 
```js
<nav>
<ul>
<li></li>
<li></li>
<li></li>
</ul>
</nav>
```
It's just a matter of semantics/accessibility.